source/managing-teams/leading-engagements.html.md

```diff
@@ -0,0 +1,88 @@
+   chunks, connecting people on different teams, or handling some bureaucracy so the engineers can
+   focus on writing code.
+3. **Finding the opportunities.** EMs should be constantly on the lookout for new opportunities to
+   create value for their clients. Perhaps a project might go faster if the team had a strong
+   frontend engineer, or there's a project on hold that the team might execute immediately. As an
+   EM, you're in a unique position: you understand the client's needs, and you also know the team's
+   capability to execute on them. The ability to use this kind of understanding to the project's
+   advantage is what makes EMs invaluable.
+
+## Supporting the team
+
+Good Engineering Managers coordinate the team to execute on the client's vision. The best EMs,
+however, go one step further and align the client's and the team's interests, so that
+**everyone feels like they're part of a single team, working together towards a shared vision.**
+
+Creating this kind of environment is more art than science, but here are some battle-tested
+practices that will help you drive the team in the right direction:
+
+- **Encourage cross-pollination.** Some consulting firms create an "us vs. them" mindset, where
+  working with clients is seen as a necessary evil. They do this by minimizing points of contact
+  and creating "gatekeepers" such as project managers. We do the opposite: we encourage the team to
+  interact with clients as much as possible, promoting public communication and direct interactions.
+  This helps the team learn more about the client's vision and industry, and feel like they're part
+  of the client's mission. 
+- **Allow people to take ownership.** As a manager, you might be tempted to own every single goal
+  and initiative your team commits to. This kind of micromanagement will alienate your team and lead
+  you to burnout. Instead, find ways to cleanly and responsibly delegate ownership to your team, and
+  limit yourself to supporting them. This will seem scary at first, but it will bring out the best
+  in your team, and it's the only sane way to scale your workload in the long term.
+- **Create space for learning.** In an ideal world, your team members will be learning something
+  new with every single initiative they undertake. While this may not always be possible, and boring
+  work will always be part of the job, you should continuously look for ways to create learning
+  opportunities for your team, by putting the right people on the right projects and creating safe
+  boundaries for experimentation and innovation. You can read more about this in
+  [Promoting Growth](/managing-teams/promoting-growth/).
+
+One aspect to keep in mind is that there's a fine balance between empowering your team and
+protecting them from external pressure and time wasting. The latter activity is commonly referred to
+as **being a shit umbrella.** Finding the right balance requires time and constant introspection,
```

Comment:
@kennyadsl I don't know, this is pretty standard terminology and any experienced manager will know what it means and will probably recognize themselves in the phrase. What do you dislike about it?

Reply:
I don't like it because it seems that everything that a manager has to filter/take responsibility for is shit, while sometimes it's just the normal flow of the work. Also, I think this term encourages the attitude of they vs us with the client, with one of us sacrificing to avoiding others taking the worst part of a client, which it's the opposite of what we say in another paragraph. 

I was looking if someone else has the same feelings and I found these two interesting posts:

- https://medium.com/@ElizAyer/the-shit-umbrella-a-rant-3c0960f92bcb
- https://medium.com/the-year-of-the-looking-glass/the-use-of-the-term-shit-umbrella-70e8e6653157

Now, I'm not saying that we shouldn't be using this term anymore in any context, I will continue using it because sometimes it's a powerful metaphor. I'm just saying that maybe here we can find a better term (which I'm not able to propose and even in the posts I linked there's no alternative). Here, I just smell a little dissonance in using this term along with the other concepts that we are expressing in the guide. No strong opinion though, we can keep using it and I won't be disappointed. 🙂

Reply:
@kennyadsl I think you might have a point, although I'm unable to come up with a gentler term that will be as easy to recognize. I think we can give this some more thought and change it if we come up with something.

Reply:
What about using a simpler ["Umbrella"](https://medium.com/@philmillman/the-umbrella-theory-of-management-6d062ccc0d88) image. It seems a metaphorical negative-free meaning.

> The latter activity is commonly referred to as **being the team's umbrella.**